src/main/resources/templates/list.html

```diff
@@ -0,0 +1,10 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>Title</title>
+</head>
+<body>
+
+</body>
+</html>
```

Comment:
The /list mapping returned a string that did not exist in templates, so I created a new list.html. Try to always have atleast a blank html for the view or the app might fail compilation.